wrappers/python/pysetup.cmake.in

```diff
@@ -1,8 +1,13 @@
 set(ENV{OPENMM_INCLUDE_PATH} "@WRAPPER_INCLUDE_DIRS@")
+set(ENV{CC} "@CMAKE_C_COMPILER@")
+set(ENV{CXX} "@CMAKE_CXX_COMPILER@")
 file(TO_NATIVE_PATH "@CMAKE_INSTALL_PREFIX@/lib" OPENMM_LIB_PATH)
 set(ENV{OPENMM_LIB_PATH} "${OPENMM_LIB_PATH}")
 message("OPENMM_LIB_PATH = " $ENV{OPENMM_LIB_PATH})
 message("OPENMM_INCLUDE_PATH = " $ENV{OPENMM_INCLUDE_PATH})
+message("CC = " $ENV{CC})
+message("CXX = " $ENV{CXX})
```

Comment:
It seemed to work fine on my Linux box, as all I got was a warning:

```
/opt/intel/composer_xe_2013.5.192/bin/intel64/icc -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DMAJOR_VERSION=6 -DMINOR_VERSION=3 -I/home/swails/src/openmm/./include -I/home/swails/src/openmm/./include/openmm -I/home/swails/src/openmm/./include/openmm/internal -I/home/swails/src/openmm/openmmapi/include -I/home/swails/src/openmm/openmmapi/include/openmm -I/home/swails/src/openmm/openmmapi/include/openmm/internal -I/home/swails/src/openmm/olla/include -I/home/swails/src/openmm/olla/include/openmm -I/home/swails/src/openmm/olla/include/openmm/internal -I/home/swails/src/openmm/serialization/include -I/home/swails/src/openmm/serialization/include/openmm -I/home/swails/src/openmm/serialization/include/openmm/internal -I/home/swails/src/openmm/plugins/amoeba/openmmapi/include -I/home/swails/src/openmm/plugins/amoeba/openmmapi/include/openmm -I/home/swails/src/openmm/plugins/amoeba/openmmapi/include/openmm/internal -I/home/swails/src/openmm/plugins/rpmd/openmmapi/include -I/home/swails/src/openmm/plugins/rpmd/openmmapi/include/openmm -I/home/swails/src/openmm/plugins/rpmd/openmmapi/include/openmm/internal -I/home/swails/src/openmm/plugins/drude/openmmapi/include -I/home/swails/src/openmm/plugins/drude/openmmapi/include/openmm -I/home/swails/src/openmm/plugins/drude/openmmapi/include/openmm/internal -I/iscratch/miniconda/2/envs/py27_1/include/python2.7 -c src/swig_doxygen/OpenMMSwig.cxx -o build/temp.linux-x86_64-2.7/src/swig_doxygen/OpenMMSwig.o
icc: command line warning #10006: ignoring unknown option '-fwrapv'
```

`-fwrapv` is a gcc flag which was passed to icc because that was used with gcc to build CPython on my computer.  My main concern with setting the Python C compiler (instead of letting distutils pick its own) is that not all machines/OSes build Python with the same compiler flags, and there may be incompatibilities that are fatal rather than a simple warning.